From 3bc889b6564b70c5113f74e8add1a47b38fce04b Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Mon, 28 Nov 2022 09:47:09 -0500
Subject: [PATCH 2/3] Issue 5544 - Increase default task TTL

Description:  Increase the Time To Live of tasks from 1 hour to 12 hours

relates: https://github.com/389ds/389-ds-base/issues/5544

Reviewed by: progier(Thanks!)
---
 ldap/servers/slapd/task.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/servers/slapd/task.c b/ldap/servers/slapd/task.c
index c0e3dd7c4..1dc4f6b28 100644
--- a/ldap/servers/slapd/task.c
+++ b/ldap/servers/slapd/task.c
@@ -48,7 +48,7 @@ static uint64_t shutting_down = 0;
 #define TASK_DATE_NAME "nsTaskCreated"
 #define TASK_WARNING_NAME "nsTaskWarning"
 
-#define DEFAULT_TTL "3600"                        /* seconds */
+#define DEFAULT_TTL "43200" /* 12 hours in seconds */
 #define TASK_SYSCONFIG_FILE_ATTR "sysconfigfile" /* sysconfig reload task file attr */
 #define TASK_SYSCONFIG_LOGCHANGES_ATTR "logchanges"
 #define TASK_TOMBSTONE_FIXUP "fixup tombstones task"
-- 
2.38.1

